writer/stdlib/stdlib.go

```diff
@@ -0,0 +1,67 @@
+package stdlib
```

Comment:
Maybe this should be an internal package? The previous implementation was not exposed.

Reply:
I think exposing the implementation allow for registry pattern in end user's implementation.

Otherwise, for example if you're building GzipHandler from a configuration file you might need something like this

```go
if implementation == "default" {
    return GzipHandlerWithOpts(CompressionLevel(configLevel)))
}else {
    return GzipHandlerWithOpts(CompressionLevel(configLevel), Implementation(getImplementation(implementation)))
}
```